terraform/environments/core-network-services/firewall-rules/development_rules.json

```diff
@@ -230,20 +230,146 @@
+  "delius_mis_dev_to_hmpps_development_ldap": {
     "action": "PASS",
-    "source_ip": "${delius-core-dev}",
+    "source_ip": "${delius-mis-dev}",
     "destination_ip": "${hmpps-development}",
     "destination_port": "389",
     "protocol": "TCP"
   },
-  "delius_mis_dev_to_hmpps_development_ldap": {
+  "delius_mis_dev_to_hmpps_development_icmp": {
     "action": "PASS",
     "source_ip": "${delius-mis-dev}",
     "destination_ip": "${hmpps-development}",
+    "destination_port": "ANY",
+    "protocol": "ICMP"
+  },
+  "hmpps_development_to_delius-mis-dev_icmp": {
+    "action": "PASS",
+    "source_ip": "${hmpps-development}",
+    "destination_ip": "${delius-mis-dev}",
+    "destination_port": "ANY",
+    "protocol": "ICMP"
+  },
+  "hmpps_development_to_delius-mis-dev_http": {
+    "action": "PASS",
+    "source_ip": "${hmpps-development}",
+    "destination_ip": "${delius-mis-dev}",
+    "destination_port": "80",
+    "protocol": "TCP"
+  },
+  "hmpps_development_to_delius-mis-dev_https": {
+    "action": "PASS",
+    "source_ip": "${hmpps-development}",
+    "destination_ip": "${delius-mis-dev}",
+    "destination_port": "443",
+    "protocol": "TCP"
+  },
+  "hmpps_development_to_delius-mis-dev_oracledb": {
+    "action": "PASS",
+    "source_ip": "${hmpps-development}",
+    "destination_ip": "${delius-mis-dev}",
+    "destination_port": "1521",
+    "protocol": "TCP"
+  },
+  "hmpps_development_to_delius-mis-dev_ad_tcp_53": {
+    "action": "PASS",
+    "source_ip": "${hmpps-development}",
+    "destination_ip": "${delius-mis-dev}",
+    "destination_port": "53",
+    "protocol": "TCP"
+  },
+  "hmpps_development_to_delius-mis-dev_ad_tcp_88": {
+    "action": "PASS",
+    "source_ip": "${hmpps-development}",
+    "destination_ip": "${delius-mis-dev}",
+    "destination_port": "88",
+    "protocol": "TCP"
+  },
+  "hmpps_development_to_delius-mis-dev_ad_tcp_135": {
+    "action": "PASS",
+    "source_ip": "${hmpps-development}",
+    "destination_ip": "${delius-mis-dev}",
+    "destination_port": "135",
+    "protocol": "TCP"
+  },
+  "hmpps_development_to_delius-mis-dev_ad_tcp_389": {
+    "action": "PASS",
+    "source_ip": "${hmpps-development}",
+    "destination_ip": "${delius-mis-dev}",
     "destination_port": "389",
     "protocol": "TCP"
   },
+  "hmpps_development_to_delius-mis-dev_ad_tcp_445": {
+    "action": "PASS",
+    "source_ip": "${hmpps-development}",
+    "destination_ip": "${delius-mis-dev}",
+    "destination_port": "445",
+    "protocol": "TCP"
+  },
+  "hmpps_development_to_delius-mis-dev_ad_tcp_464": {
+    "action": "PASS",
+    "source_ip": "${hmpps-development}",
+    "destination_ip": "${delius-mis-dev}",
+    "destination_port": "464",
+    "protocol": "TCP"
+  },
+  "hmpps_development_to_delius-mis-dev_ad_tcp_636": {
+    "action": "PASS",
+    "source_ip": "${hmpps-development}",
+    "destination_ip": "${delius-mis-dev}",
+    "destination_port": "<port>",
```

Comment:
I think this was meant to be a port number

Reply:
thanks! corrected